### ISSUE TITLE
Redo layout of run container dialogue

### ISSUE THREAD

**Issue by nicjo814** [OpenMediaVault-Plugin-Developers/openmediavault-docker-gui#17]
Make more use of available screen size (width) by using multiple (two) columns.

**Comment by Koxx3** [comment]
agree. with images/containers with more than 20 options, it takes a very long time to edit each parameter since you have to scroll, click modify - it jumps to top - , scroll down , edit the value, click validate - it jumps to top -, save. :/
